src/libraries/Directory.Build.targets

```diff
@@ -135,6 +135,11 @@
     <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Upgrade the NetStandard.Library transitive xunit dependency to avoid 1.x NS dependencies. -->
+    <PackageReference Include="NETStandard.Library" Condition="'$(IsTestProject)' == 'True' and $(TargetFrameworks.Contains('net461'))" Version="$(NetStandardLibraryVersion)" />
```

Comment:
No test project in main targets net461 as the min supported .NETFramework is net462, hence this is wrong.

Also, we already have a PackageReference to NS.Library here: https://github.com/dotnet/runtime/blob/03bc2c28cc250b1a10a5394277e3cbe5b2820bce/eng/testing/xunit/xunit.targets#L4-L6. Please update that one (i.e. remove the condition) instead of adding another one.